ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1272,8 +1300,17 @@ router bgp {{ router_bgp.as }}
 {%                 for route_target in address_family.route_targets %}
       route-target export {{ address_family.address_family }} {{ route_target }}
 {%                 endfor %}
-{%                 if address_family.route_map is arista.avd.defined %}
+{%                 if address_family.address_family in ['evpn', 'vpn-ipv4', 'vpn-ipv6'] %}
+{%                     if address_family.rcf is arista.avd.defined %}
+{%                         if address_family.vpn_route_filter_rcf is arista.avd.defined and address_family.address_family in ['vpn-ipv4', 'vpn-ipv6'] %}
+      route-target export {{ address_family.address_family }} rcf {{ address_family.rcf }} vpn-route filter-rcf {{ address_family.vpn_route_filter_rcf }}
+{%                         else %}
+      route-target export {{ address_family.address_family }} rcf {{ address_family.rcf }}
+{%                         endif %}
+{%                     endif %}
+{%                     if address_family.route_map is arista.avd.defined %}
```

Comment:
Please check if both route-map and rcf can be on EOS, and if not, let route-map "win" by check that first and then have an `elif` for rcf.

Reply:
same for import

Reply:
Both can be on EOS :)
```
router bgp 1
   vrf 1
      route-target import vpn-ipv4 rcf RT_IMPORT_AF_RCF()
      route-target export vpn-ipv4 rcf RT_EXPORT_AF_RCF()
      route-target import vpn-ipv4 route-map RT_IMPORT_AF_RM
      route-target export vpn-ipv4 route-map RT_EXPORT_AF_RM
```

Reply:
all good.